### ISSUE TITLE
Windows Binary Build artifacts "System Error" when executed (double click, powershell, and Cygwin execution) v2.1.1 through v2.2.0 

### ISSUE THREAD

**Issue by madhephaestus** [deepmind/mujoco#319]
1. Install Windows 10 fresh in VirtualBox
2. Download the latest Windows x86_64 binary release of MuJoCo
3. extract the zip on Windows 10 64 bit
4. double click simulate.exe or testspeed.exe
5. Get a series of 4 popups saying DLL's are missing

![speedtestfail](https://user-images.githubusercontent.com/1254726/171717528-72c7fbdc-ad7d-402c-ba32-b377e333c18f.png)

This issue is also effecting linking against the binary release of the library for applications.

**Comment by madhephaestus** [comment]
For reference, the binary release on Linux produces the following result when run:
```
Running 10000 steps at dt = 0.005 ...

 Simulation time      : 0.23 s
 Steps per second     : 44310
 Realtime factor      : 221.55 x
 Time per step        : 0.0226 ms

 Contacts per step    : 11.44
 Constraints per step : 41.05
 Degrees of freedom   : 27

```

**Comment by yuvaltassa** [comment]
Thanks @madhephaestus,
We'll look into this soon, leaving open. We all took some time off post open-sourcing, our response times will be back to normal in a week or two :)

**Comment by madhephaestus** [comment]
Great! Thank you all for the hard work, we all very much appreciate this library becoming open source!

**Comment by madhephaestus** [comment]
this issue is replicated in all published versions on GitHub, 

Except 2.1.0, this version works as expected.

All other versions exhibit this bug.

**Comment by saran-t** [comment]
I was under the impression that MSVC redistributable comes bundled with Windows installations, but evidently this isn't always the case.

Please try downloading and installing VC++ redist from https://www.microsoft.com/en-gb/download/details.aspx?id=48145

**Comment by madhephaestus** [comment]
that did indeed fix the issue. Why did MuJoCo work in 2.1.0 without this system depenancy?

**Comment by saran-t** [comment]
MuJoCo 2.1.0 was statically linked against MSVCRT. We stopped doing this since we don't want to ship bits of binary that we do not build ourselves.

**Comment by madhephaestus** [comment]
Is there a document of all the linked libraries that MuJoCo depends on for users that intend to make a statically linked application?

**Comment by saran-t** [comment]
Do you mean for people who are building everything from source?

**Comment by saran-t** [comment]
If you're using our official builds then it should really just be the Microsoft C runtime on Windows that's "missing".

I can check again to see if we can statically link in the Microsoft CRT again if that would help.

**Comment by madhephaestus** [comment]
That would be a big help! It would be a nice comprimise if there were one archive statically linked, and a second archive of a thin release to satisfy those that want to avoid linked in binary blobs. This issue was all about the WIndows build, but the same would be true for all 3 platforms WRT the Java bindings.

**Comment by saran-t** [comment]
See also https://docs.microsoft.com/en-us/cpp/c-runtime-library/crt-library-features

TL;DR it isn't really standard practice to link Microsoft's CRT statically into DLLs. What you might want to do is to ship the MS redistributable DLLs in your jar, see https://docs.microsoft.com/en-us/cpp/windows/redistributing-visual-cpp-files for Microsoft's guidance on this.

**Comment by madhephaestus** [comment]
yeah, i can take a whack at that. If its just the CRT, then i can just bundle and deploy that at runtime. That said a statically linked lib would be kinda great :)

**Comment by madhephaestus** [comment]
can you tell me which DLL's are needed for redistribution? https://docs.microsoft.com/en-us/cpp/windows/determining-which-dlls-to-redistribute?view=msvc-170
